swap_meet/clothing.py

```diff
@@ -1,2 +1,31 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    """
+    A subclass of Item that represents an item of clothing.
+
+    Parameters:
+    ----------
+    id : int, optional
+        Id of the item. Generated randomly by default
+    fabric : str, optional
+        Type of fabric the item is made of. Defaults to "Unknown"
+    condition : int, optional
+        An integer value representing the condition of the item, 
+        where 1 is the worst and 5 is the best. Must be between 1 and 5. 
+        Defaults to 0, indicating that no condition is unknown.
+    age : tuple, optional
+        A tuple of 3 integer representing date in format (yyyy, mm, dd).
+        Defaults to current day.
+
+    """
+
+    def __init__(self, id=None, fabric="Unknown", condition=0, age=None):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
```

Comment:
Nice work on here!

On line 23 you correctly set the default value for fabric to "Unknown".

On line 24 you properly call the initializer of the parent, and pass along the values the child wants to use.

Lastly, good work remembering to refactor the function definition to include `age` so that it can get set from the subclass